style/modules/pressModule.css

```diff
@@ -0,0 +1,37 @@
+.pressModule {
+    display: flex;
+    /* overflow: hidden; */
+    background: #f5f5f5;
+    /* width: 1080px; */
+    height: 250px;
+    width: 1200px;
+}
+
+.pressModule--img {
+    width: 35%;
+    object-fit: cover;
+
+}
+
+.pressModule--header {
```

Comment:
I have some suggestions to make the text look more like the one we had in Figma: 
- The colors are `#808080`, `#303030`, `#808080`,`#606060`, respectively
- The font sizes are 20, 21, 16, 20 (change them as you see fit)
- The header is `font-family: Libre Franklin;`